pybossa/jobs.py

```diff
@@ -1297,6 +1321,46 @@ def get_management_dashboard_stats(user_email):
     send_mail(mail_dict)
 
 
+def check_and_send_task_notifications(project_id, conn=None):
+    from pybossa.core import project_repo
+
+    project = project_repo.get(project_id)
+    if not project:
+        return
+
+    reminder = project.info.get('progress_reminder', {})
+    target_remaining = reminder.get("target_remaining")
+    email_already_sent = reminder.get("sent") or False
+    if target_remaining is None:
+        return
+
+    n_remaining_tasks = n_available_tasks(project.id)
+
+    update_reminder = False
+    if n_remaining_tasks > target_remaining and email_already_sent:
+        current_app.logger.info('incomplete tasks increase over target remaining, reset reminder to active')
```

Comment:
Could we include `n_remaining_tasks`, `target_remaining` as well as the project_id and email_address `To:` field in this log statement?

Example: "Number of incomplete tasks (n_remaining_tasks) is greater than the target remaining (target_remaining). Resetting notification to active."